zulipterminal/ui_tools/boxes.py

```diff
@@ -402,6 +405,28 @@ def reactions_view(self, reactions: List[Dict[str, Any]]) -> Any:
         except Exception:
             return ''
 
+    # Use quotes as a workaround for OrderedDict typing issue.
+    # See https://github.com/python/mypy/issues/6904.
+    def footlinks_view(self,
+                       message_links: 'OrderedDict[str, Tuple[str, int]]',
+                       ) -> Any:
```

Comment:
While we don't currently type return type of urwid widgets, I think it might be clearer if we return `None` instead of the empty string, so that we are effectively returning `Optional[Any]`. That's not worth annotating now, but if we do move to a stage of urwid widget types, `Union[str, <widget>]` will be less clear than `Union[None, <widget>]`, ie. `Optional[<widget>]`.